lib/Nitrapi/Services/Gameservers/Gameserver.php

```diff
@@ -444,4 +444,43 @@ public function changeManagedRootSetting($key, $value) {
 
         return true;
     }
+
+    /**
+     * Returns the created backups of the game server.
+     *
+     * @return array
+     */
+    public function getBackups() {
+        $url = "services/" . $this->getId() . "/gameservers/backups";
+        $backups = $this->getApi()->dataGet($url)['backups'];
```

Comment:
Add isset() for the 'backups' and if the API does not respond with backups, return an empty list (or an exception)